tests/components/shelly/conftest.py

```diff
@@ -155,9 +163,89 @@ async def coap_wrapper(hass):
     return wrapper
 
 
+@pytest.fixture
+async def rest_wrapper(hass):
+    """Setups a rest wrapper with mocked device."""
+    await async_setup_component(hass, "shelly", {})
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={"sleep_period": 0, "model": "SHSW-25", "host": "1.2.3.4"},
+        unique_id="12345678",
+    )
+    config_entry.add_to_hass(hass)
+
+    device = Mock(
+        blocks=MOCK_BLOCKS,
+        settings=MOCK_SETTINGS,
+        shelly=MOCK_SHELLY,
+        status=MOCK_STATUS_COAP,
+        firmware_version="some fw string",
+        update=AsyncMock(),
+        trigger_ota_update=AsyncMock(),
+        trigger_reboot=AsyncMock(),
+        initialized=True,
+    )
+
+    hass.data[DOMAIN] = {DATA_CONFIG_ENTRY: {}}
+    hass.data[DOMAIN][DATA_CONFIG_ENTRY][config_entry.entry_id] = {}
+    wrapper = hass.data[DOMAIN][DATA_CONFIG_ENTRY][config_entry.entry_id][
+        REST
+    ] = ShellyDeviceRestWrapper(hass, device, config_entry)
+
+    block_wrapper = hass.data[DOMAIN][DATA_CONFIG_ENTRY][config_entry.entry_id][
```

Comment:
The mock is called `rest_wrapper` but is also setting up the `block_wrapper`, if it is needed a better option is to add the `rest_wrapper` setup into the existing `block_wrapper` fixture

Reply:
done